### ISSUE TITLE
Minimap: click and drag

### ISSUE THREAD

**Issue by sdegutis** [microsoft/vscode#64459]
**Feature request:** click a spot on the minimap that's not currently visible in the editor, then drag the minimap up or down.

**Expected behavior:** the minimap selection rectangle thingy should move with your cursor.

**Actual behavior:** the thing does not move, unless you let go and re-click and hold, at which point it finally starts dragging.

Previous issues:

- #21708, closed because if the minimap is taller than your viewport, the minimap widget thingy doesn't move to where your mouse is; however this feature could still be possible when the minimap fully fits in the viewport without clipping
- #59582, closed as a duplicate of #24395 but that mentions three features which don't include this
- #22540, closed as a duplicate of #21708 but see above

**Comment by alexdima** [comment]
To verify, set `minimap.mode` to `cover` or set to `contain`.

**Comment by connor4312** [comment]
`cover` works, but `contain` only works if the minimap is longer taller than the vertical height. This feels unintuitive to me--I would expect that, in the correct rending mode, scrolling would always behave the same regardless of the size of my file.

**Comment by alexdima** [comment]
FYI after #91405, the new setting is `minimap.size` and the values `proportional`, `fit` and `fill`.
